### DISCUSSION TITLE
Lines in mean radiant temperature images

### DISCUSSION THREAD

**Discussion by bweeding** [UMEP-dev/UMEP#265] [Q&A] [answered]
When generating mean radiant temperature maps, lines are appearing. 
The code used to generate the images can be found here: 
https://github.com/bweeding/phd/blob/main/las_fusion_processing_draft3_uni.py

![image](https://user-images.githubusercontent.com/57697604/112562839-5292f500-8e2c-11eb-9a2a-2ae66843ef39.png)

**Comment by biglimp** [comment]
This is not a UMEP issue. I am moving this over to Discussions where we can continue this.

**Comment by biglimp** [comment, marked as answer]
I think this is happening when you create you DSM in Fusion. I see that you write out to an ascii grid and I suspect that you the loose precision fron decimal number to integer. Check if that is so. To solve this would be to to convert the DTM to geotiff instead.
